ZenPacks/zenoss/LinuxMonitor/modeler/plugins/zenoss/cmd/linux/os_service.py

```diff
@@ -154,9 +210,8 @@
 RE_UPSTART_SERVICE = re.compile('(?P<title>[\w\-]+(\s\([\w\/]+\))?)\s'   # service title
                                 '(?P<active_status>[\w\/]+)'             # active status
                                 '(.\sprocess\s(?P<main_pid>\d+))?')      # active status if exists
-RE_SYSTEMV_SERVICE = re.compile('(?P<title>[A-Za-z0-9_\-\.\s:]+)'
-                                '((\s|:)\(pid\s+(?P<main_pid>\d+)\))?'
-                                '\sis\s(?P<active_status>[\w\s]+)')
+# Only links that start with 'S' are running in current runlevel
+RE_SYSTEMV_SERVICE = re.compile(ur'S\d+[\w-]+\s->\s\.\./init\.d/(?P<title>.+)')
```

Comment:
Please include a brief comment about what you are matching with this.

Reply:
You have pattern in sysV you are tyring to match and those you are attempting to omit.
Please specify both of those cases.

Reply:
@alam-zenoss: Regular expressions like this can be hard to understand without more context as @pcarinhas points out. Rather than dismiss the review, why not just add context that would make it easier to understand?

```python
# Only links that start with 'S' are running in current runlevel
# Matches output of "ls -l /etc/rc${RUNLEVEL}.d/"
#   lrwxrwxrwx 1 root root 16 Oct 15  2009 K36mysqld -> ../init.d/mysqld
#   lrwxrwxrwx 1 root root 15 Oct 15  2009 S50snmpd -> ../init.d/snmpd
RE_SYSTEMV_SERVICE = re.compile(ur'S\d+[\w-]+\s->\s\.\./init\.d/(?P<title>.+)')
```